src/documentation/gtfs_to_ntfs_specs.md

```diff
@@ -271,3 +273,24 @@ follows. Note that if value is not one of `0`, `1`, `2` or `3`, then set to `0`.
 | 1 | 0 | 0 | |
 | 2 | GTFS `min_transfer_time` | GTFS `min_transfer_time` | Log a warning message if the `min_transfer_time` is empty |
 | 3 | 86400 | 86400 | |
+
+### Reading shapes.txt
+
+| NTFS file | NTFS field | Constraint | GTFS file | GTFS field | Note |
+| --- | --- | --- | --- | --- | --- |
+| geometries.txt | geometry__id | ID | shapes.txt | shape_id | All slashes `/` are removed |
+| geometries.txt | geometry__wkt | Required | shapes.txt | shape_pt_lat, shape_pt_lon, shape_pt_sequence | A WKT LINESTRING geometry is created from the 3 input fields. |
+
+### Reading frequencies.txt
```

Comment:
When I did the documentation, I had these 3 special cases:
- `src/gtfs/read.rs:954` **WARN/IGNORE** [_frequency for trip <frequency.trip_id> has same start and end time_] if the `frequency` start time and end time are the same
- `src/gtfs/read.rs:964` **WARN/IGNORE** [_frequency mapped to an unexisting trip <frequency.trip_id>_] if the `frequency` refers to a `vehicle_journey` which doesn't exist
- `src/gtfs/read.rs:975` **WARN/IGNORE** [_frequency mapped to trip <frequency.trip_id> with no stop_times_] if the `vehicle_journey` relative to the `frequency` doesn't have any `stop_time`

They are very technical but I think they should be documented one way or another (except maybe the second one which might be evident enough).